modules/qtwidgets/include/modules/qtwidgets/properties/ordinalpropertywidgetqt.h

```diff
@@ -219,18 +219,26 @@ void OrdinalPropertyWidgetQt<T, Sem>::updateFromProperty() {
     T inc = ordinal_->getIncrement();
     T val = ordinal_->get();
 
+    constexpr size_t nelem =  util::flat_extent<T>::value;
```

Comment:
CamelCase
See also code below

Reply:
maybe, I have better stuff to spend my time on then renaming local variables.